exonum/src/blockchain/schema.rs

```diff
@@ -356,7 +356,7 @@ impl<'a> Schema<&'a mut Fork> {
 
     /// Adds a new configuration to the blockchain, which will become an actual at
     /// the `actual_from` height in `config_data`.
-    pub fn commit_configuration(&mut self, config_data: StoredConfiguration) {
+    pub fn commit_configuration(&mut self, config_data: &StoredConfiguration) {
```

Comment:
it is not clear for me why we need reference there?
May be, it will be better to remove clone() at line :383

Reply:
`put` in indexes wait a value by value, not by reference

Reply:
yes, and we already got this value in `pub fn commit_configuration(&mut self, config_data: StoredConfiguration)`

Reply:
But we use `config_data` after putting it into an index in `info!` macros. Therefor we should to call `clone()` anyway. No?

Reply:
We can move this info before x)

Reply:
Yes we can. But we're talking about what we have now :-) 
BY THE WAY: I guess that if `info` would be above put, clippy would had skipped func